src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSourceContext.java

```diff
@@ -147,7 +157,17 @@ public final boolean wantForkPRs() {
     public final Set<ChangeRequestCheckoutStrategy> forkPRStrategies() {
         return forkPRStrategies;
     }
-
+    /**
+     * Returns the strategy used to notify Github of build status.
+     * @return the strategy used to notify Github of build status.
+     * @since TODO
+     */
+    public final Set<? extends AbstractGitHubNotificationStrategy> notificationStrategies() {
```

Comment:
Should be `Set<AbstractGitHubNotificationStrategy>` (or better yet a `List`) the reason you had this issue is the lack of a type specification two lines lower